src/In.ProjectEKA.HipService/Common/HttpRequestHelper.cs

```diff
@@ -33,13 +35,15 @@ public static class HttpRequestHelper
                 httpRequestMessage.Headers.Add(HeaderNames.Authorization, token);
             if (cmSuffix != null)
                 httpRequestMessage.Headers.Add("X-CM-ID", cmSuffix);
+            if (correlationId != null)
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6116067)